CODEOWNERS

```
@@ -0,0 +1,25 @@
+# Code owners of APPUiO Helm charts
+
+# As a Code owner, you are requested for Pull Request reviews
+
+# Used by VSHN as part of product offering
+/appuio/k8up              @appuio/tarazed
+/appuio/mariadb-galera    @appuio/tarazed
+/appuio/redis             @appuio/tarazed
+/appuio/generic           @appuio/tarazed
+/appuio/openshift-route   @appuio/tarazed
+/appuio/secret            @appuio/tarazed
```

```suggestion
/appuio/secret            @appuio/tarazed
/appuio/haproxy           @appuio/tarazed @anothertobi
```

We use the `/appuio/haproxy` chart in the same product offering as the `redis` and `mariadb-galera` charts. IIRC, I was involved in the initial implementation of it, so you can also add me besides `@appuio/tarazed`.

---

Thanks, will do. Would you also be interested as owner for mariadb-galera and redis as well?

---

As I don't know them very well, that doesn't make sense. The HAProxy chart existed >1.5y before the other two got forked.